src/main/java/com/theelixrlabs/healthcare/constants/DoctorConstants.java

```diff
@@ -0,0 +1,16 @@
+package com.theelixrlabs.healthcare.constants;
+
+/**
+ * Constant class containing error messages and patters related to doctor.
+ */
+public class DoctorConstants {
+    public static final String AADHAAR_ALREADY_PRESENT = "Aadhaar already present in database";
+    public static final String AADHAAR_NUMBER_PATTERN = "Aadhaar number must be only 12 digits!";
+    public static final String AADHAAR_SHOULD_BE_MANDATORY = "Aadhaar should be mandatory!";
+    public static final String CREATE_DOCTOR_END_POINT = "/doctors";
```

Comment:
POST end point should be correct
make sure it defines the purpose of API

Reply:
It was mentioned in the POC2 documentaion. So should i need to change?